manager/src/main/java/org/openremote/manager/app/ConfigurationService.java

```diff
@@ -109,8 +135,68 @@ public void saveImageFile(String path, FileInfo fileInfo) {
         try (OutputStream out = new FileOutputStream(file)) {
             out.write(CodecUtil.decodeBase64(fileInfo.getContents()));
         } catch (IOException | SecurityException exception) {
-            LOG.log(Level.WARNING, "Error when saving image in manger_config.json", exception);
+            LOG.log(Level.WARNING, "Error when saving image in manager_config.json", exception);
+        }
+
+    }
+
+    public Optional<File> getManagerConfigImage(String filename){
+        File file = getManagerConfigImagePath().resolve(filename).toFile();
+
+        // fallback to OR_CUSTOM_APP_DOCROOT
+        if(!file.isFile()){
+            file = pathPublicRoot.resolve("images").resolve(filename).toFile();
```

Comment:
Need to make sure someone can't use a back reference to navigate around the storageDir e.g. filename `../../otherfile.txt`

Reply:
L241 does that now, if you find that okay